### DISCUSSION TITLE
Crack the 5-digit code

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#675] [Galaxy Trivia] [answered]
`3 2 7 8 9` - 4 digits are correct and in the right place.
`8 2 7 3 9` - 2 digits are correct and in the right place. 2 digits are correct but in the wrong place.
`9 7 8 3 4` - 3 digits are correct but in the wrong place.
`2 3 7 6 4` - 1 digit is correct and in the right place. 3 digits are correct but in the wrong place.
`6 1 7 5 8` - 1 digit is correct and in the right place. 2 digits are correct but in the wrong place.
`2 7 3 9 6` - 1 digit is correct and in the right place. 3 digits are correct but in the wrong place.
`8 4 7 5 9` - 1 digit is correct and in the right place. 1 digit is correct but in the wrong place.
`9 1 5 8 6` - 2 digits are correct and in the right place.

**Comment by DevElkami** [comment]
51324

**Comment by Rfkgaming89** [comment]
92786

**Comment by peterbaccaro** [comment, marked as answer]
**32786**

![5 digit](https://user-images.githubusercontent.com/74666079/235991149-bd4188e8-b67c-48c6-991f-f7cbe4499dae.JPG)
